### ISSUE TITLE
Fonts Too Small

### ISSUE THREAD

**Issue by marathone** [tagspaces/tagspaces#631]
Couldn't find a way to change the font size in the interface, or am I missing something obvious? Using the Linux desktop version.

**Comment by uggrock** [comment]
In the new major version of Tagspaces we will support UI-scaling.

**Comment by marathone** [comment]
OK Thanks. Any idea of when that might be forthcoming?

**Comment by coubass** [comment]
Hi all, Tagspaces is what I've been looking for for over a decade. Well, back in mid-2000's I realized that a tag-based GUI file management would be work best for me. Found nothing back then, came across several solutions later, but none worked properly. OS X tagging was a very close call, but in switching to Mac there are other things to consider.

Although an OS-integrated system could have done better (to easily tag files as you create or download them, or as you work in a file manager), Tagspaces does exactly what I wanted. The only major drawback is scaling: I have like 30 list items on one screen in a file manager, but just some 15 in Tagspaces, and this makes it much less usable for me. Scrolling in the left panel could also be avoided with smaller text size.

So thumbs-up for the new version: this could be Gmail-like compact-cozy-comfort options, text size in preferences, or just zooming in and out with Ctrl++/Ctrl+- (I would rather use the first and the third to get rid of the blank space between items and adjust the scale).

PS Although I have no knowledge of this, I tried to edit various css properties (basically, I tried all font-size values in index.html and general.css), but managed only to alter text size of the version number next to the logo. Sorry for asking this here, but is there a chance that you point me to what I need to edit to change the text size of other elements?
